src/mixins/name-request-mixin.ts

```diff
@@ -60,7 +62,8 @@ export default class NameRequestMixin extends Mixins(DateMixin) {
    * */
   isNrValid (nr: any): boolean {
     let requestActionCDList = [NameRequestTypes.CHANGE_OF_NAME, NameRequestTypes.CONVERSION]
```

Comment:
It would be good to use the resource files for this case also (which I think is only Benefit Companies but please verify).

Reply:
Faiz's PR was ready and merged first, but his work is not complete and he's aware of your changes here, so he'll update this for coop special resolutions in his next PR.